kurtosis-devnet/justfile

```diff
@@ -44,7 +44,7 @@ op-wheel-image TAG='op-wheel:devnet': (_docker_build_stack TAG "op-wheel-target"
 KURTOSIS_PACKAGE := "github.com/ethpandaops/optimism-package"
 
 # Devnet template recipe
-devnet TEMPLATE_FILE DATA_FILE="" NAME="":
+devnet TEMPLATE_FILE DATA_FILE="" NAME="": mise
```

Comment:
I have mixed feelings here. Obviously that's a bit more correct, but on the other hand mise is a pre-requisite for all development activities in this repo, so it feels odd to just wire it here.

But maybe there's a reason why you felt that was particularly an issue here?

Reply:
Hi @sigma.

It wasn't clear to me how to do this well for the whole repo. The main readme would be where I would normally document these kind of pre-requisites, but the main readme for this repo seems to avoid any kind of instruction on setup or running which seemed intentional given the separate readmes per component. I could have added a section to every sub readme but that seemed like overkill. Similarly with the makefile, there doesn't seem to be any common dependency between targets that could be used to install mise and it seemed like it could be quite a large change to ensure that all tasks install mise.

So in the end I settled for just solving the problem that I was currently facing.

Happy to update this under your guidance so that it covers all dev activities.

Reply:
Hi @sigma, just bumping this as it's been marked stale.

Reply:
honestly, I don't think it's worth doing here / at this time.

CONTRIBUTING.md#development-quick-start has the right information for mise globally.

As for just targets, even installing the mise environment is only half the solution, as we would still need to activate the environment for the dependencies to appear in the path. Plus we don't have enough common building blocks for our justfiles (yet) to handle that centrally.